Examples/replace-types-example/Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version:5.9
```

Comment:
```suggestion
// swift-tools-version:5.10
```

We've bumped the minimum version since this PR was opened 🙂